docs/source/nodes_and_pipelines/pipeline_introduction.md

```diff
@@ -236,3 +236,19 @@ The output is as follows:
 Circular dependencies exist among these items: ['first node: <lambda>([x]) -> [y]', 'second node: <lambda>([y]) -> [x]']
 ```
 </details>
+
+### Pipeline with improperly named nodes
+Nodes named with improper characters, such as the dot notation, may exhibit strange behaviour than intended.
+
+<details>
+<summary><b>Click to expand</b></summary>
+
+```python
+pipeline([node(lambda x: x, inputs: "input1", outputs: "outputs1.kedro")])
+```
+
+The output tends to be a disconnected pipeline and / or improperly formatted Kedro structure. 
+
+This is because `.` has a special meaning internally and indicates a namespace pipeline; here, the outputs segment should be disconnected as Kedro will think that there is a "outputs1" namespace pipeline.
```

Comment:
```suggestion
This is because `.` has a special meaning internally and indicates a namespace pipeline. In the example, the outputs segment should be disconnected as the name implies there is a "outputs1" namespace pipeline.
```

Reply:
What does "outputs segment should be disconnected" mean? What should the reader do? I think this needs to be clearer.